config/owasp/suppressions.xml

```diff
@@ -78,4 +78,67 @@
     <notes>Temporary suppression for commons-fileupload-1.4.jar</notes>
     <cve>CVE-2023-24998</cve>
   </suppress>
+  <suppress>
```

Comment:
This looks like a lot of suppressions, are we sure theres no update to these deps?

Reply:
1.2 needs to be updated from master, I will do that

Reply:
Done

Reply:
Please try removing these suppressions after getting the latest changes from 1.2

Reply:
Removed suppression after merge it fixed